### DISCUSSION TITLE
Welcome to firejail Discussions!

### DISCUSSION THREAD

**Discussion by netblue30** [netblue30/firejail#4052] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rusty-snake** [comment]
### [Marking a comment as an answer](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#marking-a-comment-as-an-answer)

@ anyone, please do this if you ask in Q&A. Uhh and you can upvote ~/downvote~ comments/questions with the arrows next to it.
